api/src/pcapi/routes/public/documentation_constants.py

```diff
@@ -14,6 +14,22 @@
   _For example, if you update the stock of an event, you **should not resend the `beginningDate`** if it has not changed because, otherwise it is going to trigger the reschedule process on our side._
 """
 
+# FIELDS DESCRIPTIONS
+OFFER_STATUS_FIELD_DESCRIPTION = """
+Offer status:
+
+- `ACTIVE`: offer is validated and active. \n\n
+- `DRAFT`: offer is still draft and not yet submitted for validation - this status is not applicable to offers created via this API.\n\n
```

Comment:
offer is still a draft and not yet submitted for validation. This status is not applicable to offers created via this API.